src/meta/src/stream/source_manager/worker.rs

```diff
@@ -306,6 +306,13 @@ impl ConnectorSourceWorker {
                                     tracing::warn!(error = %e.as_report(), "error happened when finish backfill");
                                 }
                             }
+                            SourceWorkerCommand::UpdateProps(new_props) => {
+                                self.connector_properties = new_props;
```

Comment:
Do we have some validation phase for the new props to make sure the new values are valid?